services/app-api/src/testHelpers/emailerHelpers.ts

```diff
@@ -355,6 +355,7 @@ const mockUnlockedContract = (
         status: 'UNLOCKED',
         stateCode: 'MN',
         stateNumber: 4,
+        reviewStatus: 'UNDER_REVIEW',
```

Comment:
I wonder if the CI flakes are emailer specific? Because I ran into this locally this morning and was surprised it wasn't picked up in the submission backend PR

Reply:
Not sure, there is a ticket in the backlog to investigate https://jiraent.cms.gov/browse/MCR-4670